src/go/guestagent/pkg/tracker/apitracker.go

```diff
@@ -274,3 +275,7 @@ func isIPv4(addr string) (bool, error) {
 
 	return false, nil
 }
+
+func portToLower(port string) string {
+	return strings.ToLower(port)
+}
```

Comment:
Why do you create an alias for `strings.ToLower` instead of calling it directly? I thought it would include some type casting, but it just works on strings, just like `strings.ToLower`.

Also why is it called `portToLower` and not `protoToLower`?

Reply:
I only put in function for readability and maintainability(we can keep track of where it's being applied), but you are right it needs to be protoToLower.

Reply:
> I only put in function for readability

I disagree that it improves readability; it just adds another level of indirection.

> but you are right it needs to be protoToLower.

You also need to change the calling locations to match (if you don't just replace it with `strings.ToLower`).